lnd_test.go

```diff
@@ -13847,8 +13847,9 @@ func testChannelBackupRestore(net *lntest.NetworkHarness, t *harnessTest) {
 	// ann is updated?
 
 	for _, testCase := range testCases {
-		success := t.t.Run(testCase.name, func(_ *testing.T) {
-			testChanRestoreScenario(t, net, &testCase, password)
+		success := t.t.Run(testCase.name, func(t *testing.T) {
+			h := &harnessTest{t: t}
+			testChanRestoreScenario(h, net, &testCase, password)
```

Comment:
will this lose the reference to the `h.testCase` value?

Reply:
yes but it should only be necessary when using `RunTestCase` as far as i can tell